capsule_conv_layer.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, in_channels, out_channels):
                                out_channels=out_channels,
                                kernel_size=9, # fixme constant
                                stride=1,
-                               bias=False)
+                               bias=True)
```

Comment:
good find!  i was experimenting with this myself locally but didn't want to commit either way.  did you measure an accuracy improvement on MNIST with this?

Reply:
Oh, I just think if there is no BN layer in network, the bias should be add up. This strategy is work in traditional convnet. But I don't know if it is useful too in capsnet, I will test it after this model can get better accuracy. It always stuck at about 99.4% acc in my local version.